src/server/utilities/liteATITracking/viewTracking/index.ts

```diff
@@ -0,0 +1,17 @@
+export default () => {
+  const observer = new IntersectionObserver(entries => {
```

Comment:
Curious if this will work in Opera Mini tbh as its not natively supported: https://caniuse.com/intersectionobserver

We Polyfill it in our canonical usage of the view tracker: https://github.com/bbc/simorgh/blob/57ae9e93ed6185c43a7e01ed8f273ee8a65e9cdf/src/app/hooks/useViewTracker/index.jsx#L50-L51 so similar Polyfilling may need added here to? Not sure how we go about that with these `<script>` though.